### PR TITLE
fix typo in fullscreen api

### PR DESCRIPTION
**Pull request by bungcip**
the right spelling is using `screen` not `Screen` according to MDN:

https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenElement
https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullScreen
https://developer.mozilla.org/en-US/docs/Web/API/Document/exitFullscreen

### CODE REVIEW

File: js/rpg_core/Graphics.js [hunk 1]
```diff
@@ -1228,8 +1228,8 @@ Graphics._requestFullScreen = function() {
  * @private
  */
 Graphics._cancelFullScreen = function() {
-    if (document.cancelFullScreen) {
-        document.cancelFullScreen();
+    if (document.fullscreenElement) {
```
Comment (krmbn0576) [inline js/rpg_core/Graphics.js]: Thanks!

Could you fix this line to below?
`if (document.exitFullscreen) {`

File: js/rpg_core/Graphics.js [hunk 2]
```diff
@@ -1199,7 +1199,7 @@ Graphics._switchFullScreen = function() {
  * @private
  */
 Graphics._isFullScreen = function() {
-    return ((document.fullScreenElement && document.fullScreenElement !== null) ||
+    return ((document.fullscreenElement && document.fullscreenElement !== null) ||
```
Comment (krmbn0576) [inline js/rpg_core/Graphics.js]: I think this method is very confusing.
line 1202 means `Graphics._isFullScreen`,
but line 1203, 1204 means actually `Graphics._isNotFullScreen`.

Could you fix it? 🙇

File: js/rpg_core/Graphics.js (hunk 1 above)
Comment (bungcip) [reply js/rpg_core/Graphics.js]: Ok. I changed it to:
```

 if (document.exitFullscreen) { 
        document.exitFullscreen();
```
based on https://www.sitepoint.com/use-html5-full-screen-api

File: js/rpg_core/Graphics.js (hunk 2 above)
Comment (bungcip) [reply js/rpg_core/Graphics.js]: Ok. I simplified it to
```
 return document.fullscreenElement ||
           document.mozFullScreen || 
           document.webkitFullscreenElement ||
           document.msFullscreenElement;
```

**Review by bungcip — COMMENTED** [review]
Hi @krmbn0576, I already updated the code based on your comment.

File: js/rpg_core/Graphics.js (hunk 2 above)
Comment (krmbn0576) [reply js/rpg_core/Graphics.js]: @bungcip 
I'm afraid this commit don't work.
Could you fix `Graphics._switchFullScreen`?

Comment (bungcip) [reply js/rpg_core/Graphics.js]: ouch. after testing it with `npm run watch` and pressing `F4` I can confirm this commit don't work. I will change it tonight.

**Comment by bungcip** [comment]
new code in `_switchFullScreen()` has been commited

**Review by krmbn0576 — APPROVED** [review]
OK, LGTM! :-)

**Review by ghost — APPROVED** [review]
LGTM!

**Pull request merged**